downstream/modules/platform/con-controller-metrics-enhancements.adoc

```diff
@@ -0,0 +1,10 @@
+[id="con-controller-metrics-enhancements"]
+
+= Metrics enhancements
+
+Additional metrics include:
+
```

Comment:
would this list be better as a definition list? Coded as:

awx_database_connections_total:: Tracks the current number of open database connections.